cmd/ocm-backplane/upgrade/cmd.go

```diff
@@ -0,0 +1,55 @@
+package upgrade
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/openshift/backplane-cli/internal/github"
+	"github.com/openshift/backplane-cli/internal/upgrade"
+	"github.com/openshift/backplane-cli/pkg/info"
+	"github.com/spf13/cobra"
+)
+
+func long() string {
+	return strings.Join([]string{
+		"Upgrades the current backplane-cli version based on",
+		"the latest release published in GitLab available for",
```

Comment:
Release will not be available on Gitlab anymore, we can just make it something like :
`Latest version release`